app/controllers/vote.js

```diff
@@ -28,7 +28,12 @@ exports.create = async (req, res) => {
 
       // Priorities cant be longer then alternatives
       if (priorities.length > election.alternatives.length) {
-        throw new errors.InvalidPrioritiesLengthError(priorities, election);
+        throw new errors.InvalidSTVPrioritiesLengthError(priorities, election);
+      }
+
+      // If this is a normal election, then the ballot can only contain 0 or 1 priorities
+      if (election.type == 'normal' && priorities.length > 1) {
```

Comment:
Put `normal` in a const somewhere and import it.

Reply:
This seems trivial right!! But "somewhere" 🤔  The backend is really MVC, but I guess I can just slap a `const` in the model file or something. Or maybe create a utils file/folder for that one line 💯